### PR TITLE
New client API

### PR DESCRIPTION
**Pull request by asutula**
Creating this as a draft for now just to give you all a heads up...

These are the updates corresponding to the big Powergate [v1.0.0 release](https://github.com/textileio/powergate/releases/tag/v1.0.0).

~~I think most changes are done, but I still need to update the tests.~~
Ok, tests and examples are updated.

### CODE REVIEW

**Comment by asutula** [comment]
Hey guys, I marked this as ready for review, but I could use your eyes for understanding why they tests are failing to run. Looks like it just times out after a while and no tests are run. I'm not very experienced with python and related tooling, so it may be good for you all to just take this code over and get things fixed up and working the way you know how. What do you think?

**Comment by asutula** [comment]
Cool, I got it to use the python gRPC bindings that we publish whenever we release Powergate. Much cleaner, no more copying proto files into this repo and generating code.

**Comment by asutula** [comment]
Updated a couple things to match the latest Powergate release 1.1.2

File: Pipfile [hunk 1]
```diff
@@ -31,10 +30,7 @@ publish = "twine upload dist/*"
...
 integration-test = "python -m pytest tests/integration/"
-test-ffs = "python -m pytest tests/integration/test_ffs.py"
+test-api = "python -m pytest tests/integration/test_api.py"
 
 [packages]
-grpcio = "*"
-protobuf = "*"
-six = "*"
-deprecated = "*"
+grpc-powergate-client = "==1.1.2"
```
Comment (asutula) [inline Pipfile:36]: This brings along all those other dependencies

Comment (apogiatzis) [reply Pipfile:36]: Thanks for the PR @asutula we really appreciate it!!! I will take a look as well, work on the integration tests and merge asap!

Comment (apogiatzis) [reply Pipfile:36]: The PR looks great to me!! Thanks for your contribution! However, I believe there is some stuff still missing before releasing v1.0.0 to be compatible with the new API. I will merge this PR in a new branch and add what's missing in there so we can release the new version! :)

**Pull request merged**